### DISCUSSION TITLE
Can't find documentation about hermione support

### DISCUSSION THREAD

**Discussion by sipayRT** [allure-framework/allure2#2350] [Documentation]
Hi

Allure supports hermione framework (see https://github.com/allure-framework/allure-js/blob/main/packages/allure-hermione/README.md), but there is no documentation in the [frameworks section](https://allurereport.org/docs/frameworks/) about it

I can add the documentation myself if you need help with it

**Comment by cheshi-mantu** [comment]
@sipayRT hey there,
Hermione is JS/TS based FW, thus you need to check here: https://github.com/allure-framework/allure-js?tab=readme-ov-file#hermione

**Reply by miripiruni** [reply]
Hi, @cheshi-mantu! Thank you for quick response. 

I think @sipayRT meant that the tool is not in the list [in docs](https://allurereport.org/docs/frameworks/#javascript--typescript).

Can we edit the documentation to send pull request? There is github repo for it?

**Reply by cheshi-mantu** [reply]
hi @miripiruni 
not all the integrations are now in the docs at allurereport.org site, the documentation process is ongoing.

**Reply by miripiruni** [reply]
OK, thank you. Looking forward to see the final result.
